config/jobs/kubernetes-sigs/network-policy-api/network-policy-api-config.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
+  kubernetes-sigs/network-policy-api:
+  - name: pull-network-policy-api-verify
+    annotations:
+      testgrid-dashboards: sig-network-policy-api
```

Comment:
Add this to the dashboard list here https://github.com/kubernetes/test-infra/blob/42820a359b66013700063b127d665ed455cc0b80/config/testgrids/kubernetes/sig-network/config.yaml#L13